src/main/java/be/seeseemelk/mockbukkit/block/state/EnderChestMock.java

```diff
@@ -75,4 +76,11 @@ public boolean isOpen()
 		return isOpen;
 	}
 
+	@Override
+	public boolean isBlocked()
+	{
+		// TODO Auto-generated method stub
+		throw new UnimplementedOperationException();
+	}
+
```

Comment:
Does Paper have Chest and EnterChest extend from a common class to implement this?

Reply:
No, the chest implementation is different enough due to double chests.